src/components/Upload.tsx

```diff
@@ -233,6 +234,10 @@ export default function Upload({
         const sheetName = workbook.SheetNames[0]
         const worksheet = workbook.Sheets[sheetName]
         csvContent = XLSX.utils.sheet_to_csv(worksheet)
+      } else if (file.type.includes("json")) {
+        const jsonContent = JSON.parse(e.target?.result as string)
```

Comment:
Hi @MischaU8 , your assertion is blocking the app from gracefully failing. Please fix this thanks!